modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/CacheGroupMetricsWithIndexTest.java

```diff
@@ -182,34 +213,34 @@ public void testIndexCreateCountPartitionsLeft() throws Exception {
 
         ignite.cluster().active(true);
 
-        IgniteCache<Object, Object> cache1 = ignite.cache(CACHE_NAME);
+        IgniteCache<Object, Object> cache2 = ignite.cache(CACHE_NAME2);
 
         String addColSql = "ALTER TABLE " + TABLE + " ADD COLUMN " + COLUMN3_NAME + " BIGINT";
 
-        cache1.query(new SqlFieldsQuery(addColSql)).getAll();
+        cache2.query(new SqlFieldsQuery(addColSql)).getAll();
 
         for (int i = 0; i < 100_000; i++) {
             Long id = (long)i;
 
-            BinaryObjectBuilder o = ignite.binary().builder(OBJECT_NAME)
+            BinaryObjectBuilder o = ignite.binary().builder(OBJECT_NAME2)
                 .setField(KEY_NAME, id)
                 .setField(COLUMN1_NAME, i / 2)
                 .setField(COLUMN2_NAME, "str" + Integer.toHexString(i))
                 .setField(COLUMN3_NAME, id * 10);
 
-            cache1.put(id, o.build());
+            cache2.put(id, o.build());
         }
 
         MetricRegistry metrics = cacheGroupMetrics(0, GROUP_NAME).get2();
 
         GridTestUtils.runAsync(() -> {
             String createIdxSql = "CREATE INDEX " + INDEX_NAME + " ON " + TABLE + "(" + COLUMN3_NAME + ")";
 
-            cache1.query(new SqlFieldsQuery(createIdxSql)).getAll();
+            cache2.query(new SqlFieldsQuery(createIdxSql)).getAll();
 
             String selectIdxSql = "select * from information_schema.indexes where index_name='" + INDEX_NAME + "'";
```

Comment:
Will be better to rewrite it to use our system view. `INFORMATION_SCHEMA` don't support by IGNITE at all.